web/api/v1/api.go

```diff
@@ -195,6 +195,11 @@ type API struct {
 	gatherer                  prometheus.Gatherer
 }
 
+type RankingFunction interface {
```

Comment:
Let's change this to something like 

```
type Ranker interface{
Rank(promql.Series) float64
}
```

as mentioned by @codesome  in our sync :hugs: